doc/version-history.rst

```diff
@@ -0,0 +1,23 @@
+===============
+Version History
+===============
+
+.. At the time of writing the Version history/release notes are not yet standardized amongst CSCs.
+.. Until then, it is not expected that both a version history and a release_notes be maintained.
+.. It is expected that each CSC link to whatever method of tracking is being used for that CSC until standardization occurs.
+.. No new work should be required in order to complete this section.
+.. Below is an example of a version history format.
+
+Unreleased
```

Comment:
So this is interesting... you're using the version history to track changes but it's not released... so that means when we release we need to update this file? 
I think this is probably something we should discuss with Andy... it applies globally... or should.